app/src/main/java/io/intrepid/contest/screens/entrysubmission/entryimage/EntryImageActivity.java

```diff
@@ -65,6 +65,8 @@ protected void onViewCreated(Bundle savedInstanceState) {
         if (!getPackageManager().hasSystemFeature(PackageManager.FEATURE_CAMERA)) {
             chooseCameraButton.setEnabled(false);
         }
+
+        setActionBarTitle(getResources().getString(R.string.entry_image_bar_title));
```

Comment:
Not super important, but you guys can also overload setActionBar to take in a string resource so you dont have to call getResource.